src/RelatedArticle.js

```diff
@@ -48,7 +66,9 @@ export default class RelatedArticle extends Component {
             placeholder={__("Link")}
             value={this.props.item.link}
             styles={{padding: "small"}}
-            onChange={this._handleLinkChange} />
+            onChange={this._handleLinkChange} 
+            error= {error && <p style={errorStyle}>{"Link Inválido"}</p>} 
```

Comment:
Could you please make this string i18n compatible?

Reply:
No problem!